www/js/con-author.js

```diff
@@ -211,12 +354,23 @@ define([
 			*/
 		},
 
-		handleSetStudentQtyNode: function(checked){
+		handleSetStudentNode: function(checked){
 			console.log("********************* in handle set student quantity node", checked);
-		},
-
-		handleSetStudentEqNode: function(checked){
-			console.log("********************* in handle set student equation node", checked);
+			if(checked){
+				style.set('modelSelectorContainer', 'display', 'block');
+				var studentNode = this._model.student.getNodeIDFor(this.currentID);
+				if(studentNode == null){
+					//to do : to write addStudentNode which has to add a quantity node, pushing this it backlog 
+					//this.addStudentNode(this.currentID);
```

Comment:
The to do is supposed to be written like TODO, as this gets highlighted in every code editor be it sublime or vim. For example check the main.js.